src/V3EmitMk.cpp

```diff
@@ -207,6 +211,19 @@ class EmitMk {
         of.puts("# Include global rules\n");
         of.puts("include $(VERILATOR_ROOT)/include/verilated.mk\n");
 
+        const std::map<std::string, V3HierarchyBlockOption>& hierOptions = v3Global.opt.hierBlocks();
+        std::vector<std::string> subLibPaths;
+        if (!v3Global.opt.hierMode()) {  // Only in top module
+            for (std::map<std::string, V3HierarchyBlockOption>::const_iterator it = hierOptions.begin();
+                 it != hierOptions.end(); ++it) {
+                const std::string& mangledName = it->second.mangledName();
+                const std::string subPrefix = "V" + mangledName;
+                const std::string subLibPath = subPrefix + "/lib" + mangledName + ".a";
+                of.puts(subLibPath + ":\n");
+                of.puts("\t${MAKE} -C " + subPrefix + " -f " + subPrefix + ".mk lib" + mangledName + ".a VM_PREFIX=V" + mangledName + "\n");
```

Comment:
Maybe this also uses the buildArguments function I mentioned elsewhere?

Reply:
When calling subprocess, it should refer `v3Global.opt.getenvMAKE()`.
Now it is factored  out as the following commit 742d3e0112780d802ff157feb965f6798a4958ae .

In the Makefile, `${MAKE}` should be `${MAKE}` because the variable is special for parallel execution as the [document](https://www.gnu.org/software/make/manual/html_node/MAKE-Variable.html#MAKE-Variable) says.